aces/Loris
# Move feedback MRI comments popup code out of htdocs into the imaging_browser module code

## Brief summary of changes

This moves the feedback MRI comments popup code out of the `htdocs` directory to have it in the `imaging_browser` module code instead .

#### Testing instructions (if applicable)

1. Checkout the code from this branch
2. Run `make` or `make dev`
3. Open an imaging session in the imaging browser module and click on the `QC Comments` button below a given image
4. Test that the information displayed is correct and that it saves new data when user has qc permission
5. In the imaging session page of the imaging browser, click on the `Visit Level Feedback` button on the left Navigation panel and make sure the information displayed is correct and that it saves new data when user has qc permission

#### Link(s) to related issue(s)

Related to https://github.com/aces/HBCD/issues/225 (Step 1 of the issue)
